### PR TITLE
Exclusion Windows resource was transferred to the APIv2

### PR DESCRIPTION
**Pull request by testisnullus**

### CODE REVIEW

File: apis/clusterresources/v1beta1/exclusionwindow_types.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	return client.MergeFrom(old)
+}
+
+func (ew *ExclusionWindow) ValidateExclusionWindow() error {
+	if !validation.Contains(ew.Spec.DayOfWeek, models.DaysOfWeek) {
+		return fmt.Errorf("%v, available values: %v",
+			models.ErrIncorrectDayOfWeek, models.DaysOfWeek)
+	}
+	return nil
+}
```
Comment (ribaraka) [inline apis/clusterresources/v1beta1/exclusionwindow_types.go:83]: I suggest you move validation funcs to webhook file where all validations are written

File: controllers/clusterresources/exclusionwindow_controller.go [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+			ew, models.Normal, models.Created,
+			"Resource creation request is sent. Resource ID: %s",
+			id,
+		)
+
+		patch := ew.NewPatch()
+		ew.Status.ID = id
+		err = r.Status().Patch(ctx, ew, patch)
+		if err != nil {
+			l.Error(err, "cannot patch Exclusion Window resource status",
```
Comment (ribaraka) [inline controllers/clusterresources/exclusionwindow_controller.go:127]: please add here more info, like "cannot patch...after creation"

**Review by ribaraka — CHANGES_REQUESTED** [review]

File: controllers/clusterresources/exclusionwindow_controller.go (hunk 2 above)
Comment (testisnullus) [reply controllers/clusterresources/exclusionwindow_controller.go]: Fixed

File: apis/clusterresources/v1beta1/exclusionwindow_types.go (hunk 1 above)
Comment (testisnullus) [reply apis/clusterresources/v1beta1/exclusionwindow_types.go]: Fixed

**Review by OleksiienkoMykyta — APPROVED** [review]

**Review by ribaraka — APPROVED** [review]

**Pull request merged**